### PR TITLE
Bug fixes for GUI and Websocket connections

### PR DESCRIPTION
**Pull request by robertbartel**
- Fix typo in `nwm_gui` Docker stack `app_server` service volume config
- Fix bug with WebSocketInterface related to extending MaaSRequest class to ModelExecRequest 
- Bumping Python package deps and versions
- Updating version of `dmod.communication` package required by GUI code

### CODE REVIEW

**Comment by mattw-nws** [comment]
Am I missing it or are all the dependency bumps OWP software? No external dependencies actually changed?

**Comment by robertbartel** [comment]
This PR probably needs to be merged prior to final review of #130.

**Comment by robertbartel** [comment]
@mattw-nws, correct.  These are internal dependency relationships between the different namespace packages.  There is just a bit of a domino effect.

As a brief aside: the package architecture probably needs to be revisited to avoid the dependency complexity we are seeing right now, but that will have to be addressed later.

File: python/services/requestservice/setup.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
     author_email='',
     url='',
     license='',
-    install_requires=['websockets', 'dmod-communication>=0.4.1', 'dmod-access>=0.1.1',
+    install_requires=['websockets', 'dmod-communication>=0.4.2', 'dmod-access>=0.2.0',
```
Comment (mattw-nws) [inline python/services/requestservice/setup.py:17]: Notably [/python/lib/access/dmod/access/_version.py](https://github.com/robertbartel/DMOD/blob/stack_fixes/websockservice/python/lib/access/dmod/access/_version.py) still shows `0.1.1`, so is this an error or are there missing changes in dmod-access, perhaps?

**Review by mattw-nws — CHANGES_REQUESTED** [review]

**Review by mattw-nws — APPROVED** [review]

**Pull request merged**